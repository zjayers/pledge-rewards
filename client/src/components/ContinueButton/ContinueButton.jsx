import React from 'react';
import Button from './ContinueButton.style';

const ContinueButton = () => {
  return <Button>Continue</Button>;
};

export default ContinueButton;
