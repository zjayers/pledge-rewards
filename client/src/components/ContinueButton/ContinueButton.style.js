import styled from 'styled-components';

export default styled.button`
  width: 100%;
  background-color: #009e74;
  color: #ffffff;
  height: 4.2rem;
  padding: 0 1.8rem;
  font-size: 1.4rem;
  font-weight: 500;
  transition: all 0.25s ease-in-out !important;
  text-align: center;
  cursor: pointer;
  border: none;
  position: relative;
  display: inline-block;
  border-radius: 0;
  margin-top: 18px;

  &:hover:not(:disabled) {
    background-color: #037362;
  }
  
  &:hover:not(:disabled) {
    transform: translateY(-1px);
  }
  
  &:focus {
    outline: 1px solid #282828;
    outline-offset: 2px;
    transition: outline 0.25s ease-in-out;
`;
