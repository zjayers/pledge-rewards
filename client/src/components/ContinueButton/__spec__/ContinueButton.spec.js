import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import ContinueButton from '../ContinueButton';

describe('ContinueButton Component', () => {
  let component;

  beforeEach(() => {
    component = mount(<ContinueButton />);
  });

  afterEach(() => {
    component.unmount();
  });

  it('should render without failure', () => {
    expect(component).toBeDefined();
  });

  it('should match the test snapshot', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
