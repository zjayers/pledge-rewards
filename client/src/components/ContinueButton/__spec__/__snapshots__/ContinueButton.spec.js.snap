// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContinueButton Component should match the test snapshot 1`] = `
<ContinueButton>
  <ContinueButtonstyle___default>
    <button
      className="ContinueButtonstyle___default-mwgchk-0 hTA-dXx"
    >
      Continue
    </button>
  </ContinueButtonstyle___default>
</ContinueButton>
`;
